db/install.xml

```diff
@@ -1,14 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<XMLDB PATH="blocks/opencast/db" VERSION="20180704" COMMENT="XMLDB file for Moodle blocks/opencast"
+<XMLDB PATH="blocks/opencast/db" VERSION="20190718" COMMENT="XMLDB file for Moodle blocks/opencast"
```

Comment:
You need to repeat all changes within the install.xml also in the upgrade.php in order to get it to work for sites, which already have a version of block_opencast installed. Through the XMLDB Editor you can generate the code for the upgrade.php.